app/jobs/enrol_school_cohorts_job.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class EnrolSchoolCohortsJob < ApplicationJob
+  def perform
+    SchoolCohort
```

Comment:
don't bother if workload is small but you can eager load `school` as a minimum, there may be others